app/assets/stylesheets/partials/_base.scss

```diff
@@ -91,4 +91,14 @@ Welsh/English language selector
 }
 .translate + .content__body h1 {
   margin-top: 0.8em;
-}
\ No newline at end of file
+}
+
+/*
+====================================
+User research link is on blue background so we
+don't want the visited colour to be blue too.
+====================================
+*/
+.notice-banner__wrapper a:visited {
+    color: white;
```

Comment:
Use hex-code for consistency.
This should also sit with the rest of the panel code in the partial

Reply:
@adamliptrot-oc is this something that still needs to be amended?